### ISSUE TITLE
Setup Failed - 0x80072efd - Unspecified Error - No Internet Causes Problems?

### ISSUE THREAD

**Issue by spottedmahn** [dotnet/core-setup#2438]
## Steps to reproduce
Try to install on a machine w/o internet that needs VC_redist.x64.exe

## Expected  behavior
A better error message.  Help on how to get around this.

## Actual behavior
![image](https://cloud.githubusercontent.com/assets/14079228/26216489/4c1ce786-3bd2-11e7-8fc9-e57220750abc.png)

[Microsoft_.NET_Core_1.1.2_-_Runtime_(x64)_20170518133910.log.txt](https://github.com/dotnet/core-setup/files/1011840/Microsoft_.NET_Core_1.1.2_-_Runtime_.x64._20170518133910.log.txt)

## Environment data
Windows Server 2012 R2

**Comment by rakeshsinghranchi** [comment]
Thanks for reporting this issue. In .NET Core 2.0.0 , runtime installer does not chain VC_redist.x64.exe and thus, this issue does not occur there.  
Please let me know if you are still blocked on installing 1.1.2 runtime.

**Comment by rakeshsinghranchi** [comment]
.NET 2.0.0 setup doesn't download VC_redist . This doesn't meet the servicing bar for 1.1.2 or previous versions.Hence,closing it. Please open a separate issue if you are blocked on installing 1.1.2 runtime .

**Comment by spottedmahn** [comment]
@rakeshsinghranchi it was blocking my 1.1.2 install but thankful I figured that out from the log file and I manually downloaded VC_redist and installed it.  After VC_redist install the .net core 1.1.2 install went fine.  Thanks!
